railties/test/application/middleware/session_test.rb

```diff
@@ -407,6 +415,8 @@ def read_raw_cookie
       add_to_config <<-RUBY
         secrets.secret_token = "3b7cd727ee24e8444053437c36cc66c4"
         Rails.application.credentials.secret_key_base = nil
+
+        config.action_dispatch.use_cookies_with_metadata = false
```

Comment:
I'm a little unsure about all these tests that disable metadata. It would seem more right to me that our own tests are updated and then we add some specific tests for the backwardscompatibility.